packages/extension/webpack.config.cjs

```diff
@@ -98,4 +99,35 @@ const popup = {
   ...commonConfig,
 };
 
-module.exports = [root, devTools, popup];
+const report = {
+  entry: {
+    index: './src/view/report/index.tsx',
+  },
+  output: {
+    path: path.resolve(__dirname, '../../dist/extension/report'),
+    filename: '[name].js',
+  },
+  devServer: {
+    static: {
+      directory: path.join(__dirname, './dist'),
+    },
+    compress: true,
+    port: 9000,
+  },
```

Comment:
What is the use of creating a dev server with port 9000 for the static report?

Reply:
I was trying for this React app to be able to developed in a browser. Removing it now.